pom.xml

```diff
@@ -4,7 +4,7 @@
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
 
-        <version>1.509.4</version>
+        <version>1.609.1</version>
```

Comment:
Needs a review from plugin maintainers. I'm not sure which baselines do they target